releasetasks/templates/l10n.yml.tmpl

```diff
@@ -135,6 +135,15 @@
             {% if running_tests is defined %}
             task_name: "{{ '{}_beetmover_candidates_{}'.format(buildername, chunk) }}"
             {% endif %}
+            build_props:
+                locales:
+{% for locale in our_locales %}
+                    - {{ locale }}
```

Comment:
The tasks are immutable. In case we fail with repacks, we would start another build from scratch. This doesn't include automatic reruns due to infra or other issues.